File: ABC/301-350/350/b.cpp
#include <bits/stdc++.h>
using namespace std;
#define rep(i,n) for(int i=0;i<(n);i++)

int main(){
    int n,q;
    cin >> n >> q;
    vector<bool> exist(n,true);
    rep(qi,q){
        int t;
        cin >> t;
        t--;
        if(exist[t]) exist[t] = false;
        else exist[t] = true;
    }
    int ans = 0;
    rep(i,n){
        if(exist[i]) ans++;
    }
    cout << ans << endl;
    return 0;
}

File: ABC/301-350/350/b.py
n,q = map(int,input().split())
exist = [True]*n
t = list(map(int,input().split()))
for i in range(q):
    nt = t[i]
    nt -= 1
    if exist[nt]:
        exist[nt] = False
    else:
        exist[nt] = True

ans = 0
for i in range(n):
    if exist[i]:
        ans += 1

print(ans)